crates/rooch-types/src/transaction/mod.rs

```diff
@@ -174,15 +172,15 @@ pub struct TransactionInfo {
     pub status: KeptVMStatus,
 }
 
-impl TransactionInfo {
+impl TransactionExecutionInfo {
```

Comment:
transaction storage needs to use TransactionExecutionInfo, but TransactionExecutionInfo belongs to rooch types, consider putting TransactionExecutionInfo into moveos type, otherwise moveos will be reversely dependent on rooch.

Reply:
I also think about this. How to store the TransactionExecutionInfo and TransactionSequenceInfo.

I try to figure out a solution on the next PR.

Reply:
from my understanding, the entry for calling Sequence and calling Transaction storage should also be in the rooch executor, right?

Reply:
Currently, the RPC service calls executor and sequencer.